update.sh

```diff
@@ -118,13 +118,18 @@ function create_variant() {
       install_type='develop'
     fi
 
+  # Get Year / Month of the Version string
```

Comment:
Code standards: The indenting style of this file is all over the place, please decide on tabs or spaces.